ports/esp32s2/boards/artisense_rd00/board.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Scott Shawcroft for Adafruit Industries
```

Comment:
Please update these files with your own copyright information.  E.g., "Copyright (c) 2021 <your name>", adding "for <company name>" if applicable.